resources/aws_ec2.go

```diff
@@ -20,25 +20,49 @@ package resources
 import (
 	"context"
 	"encoding/base64"
+	"encoding/json"
 	"fmt"
 	"log"
+	"net/http"
+	"strconv"
 	"sync"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/request"
 	"github.com/aws/aws-sdk-go/aws/session"
+	cwe "github.com/aws/aws-sdk-go/service/cloudwatchevents"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/sns"
+	multierr "github.com/hashicorp/go-multierror"
 	errwrap "github.com/pkg/errors"
 )
 
 func init() {
 	RegisterResource("aws:ec2", func() Res { return &AwsEc2Res{} })
 }
 
-// AwsPrefix is a const which gets prepended onto the instance name.
-const AwsPrefix = "mgmt:"
+const (
+	// AwsPrefix is a const which gets prepended onto object names.
+	AwsPrefix = "_mgmt-"
```

Comment:
I kind of liked the colon. Can we use that instead of a dash? Or is it better. Everything else looks great!

FYI: one liner comments don't need to be full sentences, but it's okay if they are. No changes needed.

Reply:
No colons allowed in topic names (or cloudwatch rules afaict) :( Tried slashes and pipes too.

Reply:
SO *that's* valuable information that should be in a source comment!

Reply:
gotcha